modules/cert-manager-acme-dns01-explicit-azure.adoc

```diff
@@ -0,0 +1,163 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-dns01-explicit-azure_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges by using explicit credentials for Microsoft Azure DNS
+
+You can use {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using explicit credentials on Microsoft Azure. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Azure DNS.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have set up an IAM role for Azure DNS. For more information, see link:https://cert-manager.io/docs/configuration/acme/dns01/azuredns/[Azure DNS] in the upstream cert-manager documentation.
```

Comment:
> You have set up an IAM role for Azure DNS.

Incorrect. For Azure, it is not only IAM role, it is service principal https://cert-manager.io/docs/configuration/acme/dns01/azuredns/#service-principal of right role.